packages/lending/src/protocols/radiant-v2/configs.ts

```diff
@@ -29,6 +29,13 @@ export const configs: Config[] = [
       PriceOracle: '0x0BB5c1Bc173b207cBf47CDf013617087776F3782',
     },
   },
+  {
+    chainId: common.ChainId.base,
```

Comment:
please test getPortfolio on the new added base chain in lending-protocol.test.ts